omegaconf/omegaconf.py

```diff
@@ -502,13 +502,16 @@ def to_container(
         cfg: Any,
         *,
         resolve: bool = False,
+        throw_on_missing: bool = False,
         enum_to_str: bool = False,
         exclude_structured_configs: bool = False,
     ) -> Union[Dict[DictKeyType, Any], List[Any], None, str]:
         """
         Resursively converts an OmegaConf config to a primitive container (dict or list).
         :param cfg: the config to convert
         :param resolve: True to resolve all values
+        :param throw_on_missing: Whether an error should be thrown when "???"
+                values are encountered in the given config object.
```

Comment:
Side note:
Currently we do not generate API docs.
It would actually be useful as the API surface area increases.

Reply:
Created issue #536 to keep track of this idea.